_posts/2019-09-05-eks-openfaas-cloud-build-guide.md

```diff
@@ -343,18 +349,18 @@ You can also [install your own self-hosted registry](https://github.com/alexelli
 
 > Note: using AWS ECR is beyond the scope of this tutorial. Contributions are welcome, see also: [OFC: adding support for AWS ECR](https://github.com/openfaas/openfaas-cloud/issues/504).
 
-When you've decided which works for you, check your Docker Desktop or Docker client settings to make sure the password is *not* stored in the keychain.
+Now log in, using ofc-bootstrap and check the `credentials/config.json` file, the credentials should be encoded in base64.
 
-![](/images/2019-aws-ofc/no-key-chain-storage.png)
-
-Now log in, and check the `~/.docker/config.json` file, the credentials should be encoded in base64.
-
-My registry is `registry.on-k3s.dev` and my user is: `ofcuser`, therefore I ran:
+My registry is `registry.on-k3s.dev` my user is: `ofcuser` and my password is: `my-password`, therefore I ran:
 
 ```sh
-docker login registry.on-k3s.dev --username ofcuser
+ofc-bootstrap registry-login --registry-url registry.on-k3s.dev \ 
```

Comment:
Given that `registry` is already in the command, maybe we should update the Go code to use `--url` instead? or the more common `--server`?

Reply:
This is very true - I will update that, then this pr once that's done